roles/cloud-cloudstack/tasks/venv.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Clean up the environment
+  file:
+    dest: "{{ cloudstack_venv }}"
+    state: absent
+  when: clean_environment
+
+- name: Install requirements
+  pip:
+    name:
+      - cs
+      - sshpubkeys
```

Comment:
What is `sshpubkeys` for?

Reply:
it's a requirement from the `cs_sshkeypair` module